spec.md

```diff
@@ -1657,40 +1629,35 @@ If a `CreateSnapshot` operation times out before the snapshot is cut, leaving th
 2. The CO takes no further action regarding the timed out RPC, a snapshot is possibly leaked and the operator/user is expected to clean up.
 
 It is NOT REQUIRED for a controller plugin to implement the `LIST_SNAPSHOTS` capability if it supports the `CREATE_DELETE_SNAPSHOT` capability: the onus is upon the CO to take into consideration the full range of plugin capabilities before deciding how to proceed in the above scenario.
-A controller plugin COULD implement the `LIST_SNAPSHOTS` capability and call it repeatedly with the `snapshot_id` as a filter to query whether the uploading process is complete or not if it needs to upload a snapshot after it is being cut.
-
-##### Snapshot Statuses
 
-A snapshot could have the following statusus: UPLOADING, READY, and ERROR.
+##### The is_ready_to_use Parameter
 
-Some cloud providers will upload the snapshot to a location in the cloud (i.e., an object store) after the snapshot is cut.
-Uploading may be a long process that could take hours.
-If a `freeze` operation was done on the application before taking the snapshot, it could be a long time before the application can be running again if we wait until the upload is complete to `thaw` the application.
+Some cloud providers will process the snapshot after the snapshot is cut, i.e., uploading the snapshot to a location in the cloud (i.e., an object store) after the snapshot is cut.
+A process such as uploading may be a long process that could take hours.
+If a `freeze` operation was done on the application before taking the snapshot, it could be a long time before the application can be running again if we wait until the process is complete to `thaw` the application.
 The purpose of `freeze` is to ensure the application data is in consistent state.
 When `freeze` is performed, the container is paused and the application is also paused.
 When `thaw` is performed, the container and the application start running again.
-During the snapshot uploading phase, since the snapshot is already cut, a `thaw` operation can be performed so application can start running without waiting for the upload to complete.
-The status of the snapshot will become `READY` after the upload is complete.
+During the snapshot processing phase, since the snapshot is already cut, a `thaw` operation can be performed so application can start running without waiting for the process to complete.
+The `is_ready_to_use` parameter of the snapshot will become `true` after the process is complete.
 
-For cloud providers and storage systems that don't have the uploading process, the status should be `READY` after the snapshot is cut.
-`thaw` can be done when the status is `READY` in this case.
+For cloud providers and storage systems that don't have the process, the `is_ready_to_use` parameter should be `true` after the snapshot is cut.
```

Comment:
Find and replace all instances of
`should` -> `SHOULD`
`may` -> `MAY`
`must` -> `MUST`

At least the bits you are touching in this PR. I will open a PR addressing the rest of the spec.

Reply:
Done